ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -238,39 +238,113 @@ spec:
+        value = '0'
+    );
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS2.3.3.1
-  contributors: 
+  contributors: artemist-work
 ---
 apiVersion: v1
 kind: policy
 spec:
   name: CIS - Ensure Screen Sharing Is Disabled 
   platforms: macOS
   platform: darwin
-  description: 
-  resolution: ""
-  query: SELECT * from time;
+  description: |
+    Screen Sharing allows a computer to connect to another computer on a network and
+    display the computer’s screen. While sharing the computer’s screen, the user can
+    control what happens on that computer, such as opening documents or applications,
+    opening, moving, or closing windows, and even shutting down the computer.
+    Disabling Screen Sharing mitigates the risk of remote connections being made without
+    the user of the console knowing that they are sharing the computer.
+  resolution: |
+    Graphical Method:
+    1. Open System Settings
+    2. Select General
+    3. Select Sharing
+    4. Set Screen Sharing to disabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT * FROM plist WHERE
+        path = '/var/db/com.apple.xpc.launchd/disabled.plist' AND
+        key = 'com.apple.screensharing' AND
+        value = '0'
+    );
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS2.3.3.2
-  contributors: 
+  contributors: artemist-work
 ---
 apiVersion: v1
 kind: policy
 spec:
   name: CIS -  Ensure File Sharing Is Disabled
   platforms: macOS
   platform: darwin
-  description: 
-  resolution: ""
-  query: SELECT * from time;
+  description: |
+    File sharing from a user workstation creates additional risks by
+    increasing complexity and making security more difficult. Hardened
+    file servers should be used instead of workstations
+  resolution: |
+    Graphical Method:
+    1. Open System Settings
+    2. Select General
+    3. Select Sharing
+    4. Set File Sharing to disabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
+      SELECT * FROM plist WHERE
+        path = '/var/db/com.apple.xpc.launchd/disabled.plist' AND
+        key = 'com.apple.smbd' AND
+        value = '0'
+    );
   purpose: Informational
   tags: compliance, CIS, CIS_Level1, CIS2.3.3.3
-  contributors: 
+  contributors: artemist-work
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS -  Ensire Remote Login Is Disabled
+  platforms: macOS
+  platform: darwin
+  description: |
+    Remote Login allows an interactive terminal session to a computer.
+    The SSH server built into macOS should not be enabled on a standard user computer,
+    particularly one that changes locations and IP addresses.
+    A standard user that runs local applications, including email, web browser,
+    and productivity tools, should not use the same device as a server
+
+  resolution: |
+    Graphical Method:
+    1. Open System Settings
+    2. Select General
+    3. Select Sharing
+    4. Set Remote Login to disabled
+  query: |
+    SELECT 1 WHERE NOT EXISTS (
```

Comment:
I used `#` for comments because it looked cleaner to me. I explained in 2.3.3.1 but the explanation is the same for everything else.

Reply:
Returns `unrecognized token: "#"` when running the query on hosts.

If you prefer to use `#`,  let's move the comments to the "yaml" side instead of in the string.
And let's add the same comment on the three checks so it's easy to spot by future readers.

Reply:
`#` is a yaml comment within a string, fleetctl apply still works for me

Reply:
Yep, `fleetctl apply` works, it's the SQL query execution (in osquery) that is not working.

Reply:
I changed the indentation and I think it's working now, can you test?

Reply:
Cool. Looks good. You can test by applying the yaml and then go to Fleet and run the policy on a macOS device.

Reply:
It works for me, I just wanted a second opinion.

Reply:
Yeah, it looks good to me, in any case @xpkoala will QA this shortly.